recipes-browser/wpewebkit/wpewebkit.inc

```diff
@@ -22,6 +22,7 @@ TOOLCHAIN = "gcc"
 PACKAGECONFIG ??= "fetchapi indexeddb mediasource video webaudio webcrypto woff2 gst_gl remote-inspector openjpeg"
 
 # WPE features
+PACKAGECONFIG[accessibility] = "-DENABLE_ACCESSIBILITY=ON,-DENABLE_ACCESSIBILITY=OFF,atk"
```

Comment:
at-spi2-core isn't needed there? I had to have it when i tried to enable a11y with a trunk snapshot.

Reply:
done